backend/product/src/main/java/org/ecom/product/domain/Picture.java

```diff
@@ -9,10 +13,16 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
+@Entity
 public class Picture {
+
+  @Id
+  @GeneratedValue
   private Long id;
   private Integer width;
   private Integer height;
   private String path;
-  private Integer order;
```

Comment:
This order param does not allow Hibernate to create the SQL Tables and won't be able to create Tables from Entities.